### PR TITLE
Adding Quac!

### PR DESCRIPTION
**Pull request by rdefeo**
# Application Submission

- A QUick ACtion remote control app that lets you send saved signals of varying types. After recording signals using the existing firmware applications, which are then saved on the SD card, users will re-organize those files into the `/ext/apps_data/quac` folder to generate the app's UI. This lets users logically group actions, even though they may be of different types. For example, you could create a "Work" group/folder which contains a sub-ghz signal for the parking gate, and a rfid signal for opening the parking structure door. Users no longer need to perform lengthy navigation to each ap to find their saved signals.

# Extra Requirements 

- Does not require extra hardware or software

# Author Checklist (Fill this out)

- [x] I've read the [contribution guidelines](../blob/HEAD/documentation/Contributing.md) and my PR follows them
- [x] I own the code I'm submitting or have code owner's permission to submit it
- [ ] I [have validated](../blob/HEAD/documentation/Contributing.md#validating-manifest) the manifest file(s) with `python3 tools/bundle.py --nolint applications/CATEGORY/APPID/manifest.yml bundle.zip`

# Reviewer Checklist (Don't fill this out)

- [ ] Bundle is valid
- [ ] There are no obvious issues with the source code
- [ ] I've ran this application and verified its functionality

### CODE REVIEW

File: applications/Tools/quac/manifest.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+sourcecode:
+  type: git
+  location:
+    origin: https://github.com/rdefeo/quac.git
+    commit_sha: b884fea5a669ba3ca8b3536d9b27df9f54fafce4
+description: "@./README.md"
+changelog: "@./CHANGELOG.md"
```
Comment (hedger) [inline applications/Tools/quac/manifest.yml:7]: This file does not exist in the specified git revision.

Please fix it and run validation script.

Comment (rdefeo) [reply applications/Tools/quac/manifest.yml:7]: Updated the manifest.yml to the latest git revision commit_sha, and re-ran the validation script!

Comment (hedger) [reply applications/Tools/quac/manifest.yml:7]: So, what was the validation script's output?

I see 
`Error: 09:59:24.031 [E] Icon quac.png is not black and white`

Comment (rdefeo) [reply applications/Tools/quac/manifest.yml:7]: Working on changes - I completely missed this error, my apologies. And even though my icon looked fine on the Flipper and in image apps, when I dumped the raw data, the whites were actually 255, 254, 254 <smacks head on desk>

Updating to latest firmware and addressing some other issues I recently found. I appreciate your patience - thank you

**Review by doomwastaken — APPROVED** [review]

**Pull request merged**